core/arch/x86/encode.c

```diff
@@ -868,24 +882,29 @@ reg_size_ok(decode_info_t *di /*prefixes field is IN/OUT; x86_mode is IN*/, reg_
         opnd_size_t expanded = expand_subreg_size(opsize);
         if (expanded == OPSZ_8 &&
             (optype == TYPE_P || optype == TYPE_Q || optype == TYPE_P_MODRM))
-            return (reg >= REG_START_MMX && reg <= REG_STOP_MMX);
+            return (reg >= DR_REG_START_MMX && reg <= DR_REG_STOP_MMX);
```

Comment:
These seem like an unrelated cleanup for a different PR.

Reply:
Reverted.